org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/contentmodel/DTDDiagnosticsTest.java

```diff
@@ -767,6 +767,73 @@ public void diagnosticRelatedInformationWithXMLModel() throws Exception {
 				diagnostic, diagnosticBasedOnDTD);
 	}
 
+	@Test
+	public void defaultEntityExpansionLimit() {
+		ContentModelSettings settings = new ContentModelSettings();
+		settings.setUseCache(true);
+		XMLValidationSettings validationSettings = new XMLValidationSettings();
+		validationSettings.setResolveExternalEntities(true);
+		settings.setValidation(validationSettings);
+		
+		String xml = "<?xml version=\"1.0\"?>\r\n" + //
+				"<!DOCTYPE lolz [\r\n" + //
+				"    <!ENTITY lol \"lol\">\r\n" + //
+				"    <!ELEMENT lolz (#PCDATA)>\r\n" + //
+				"    <!ENTITY lol1 \"&lol;&lol;&lol;&lol;&lol;&lol;&lol;&lol;&lol;&lol;\">\r\n" + //
+				"    <!ENTITY lol2 \"&lol1;&lol1;&lol1;&lol1;&lol1;&lol1;&lol1;&lol1;&lol1;&lol1;\">\r\n" + //
+				"    <!ENTITY lol3 \"&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;&lol2;\">\r\n" + //
+				"    <!ENTITY lol4 \"&lol3;&lol3;&lol3;&lol3;&lol3;&lol3;&lol3;&lol3;&lol3;&lol3;\">\r\n" + //
+				"    <!ENTITY lol5 \"&lol4;&lol4;&lol4;&lol4;&lol4;&lol4;&lol4;&lol4;&lol4;&lol4;\">\r\n" + //
+				"    <!ENTITY lol6 \"&lol5;&lol5;&lol5;&lol5;&lol5;&lol5;&lol5;&lol5;&lol5;&lol5;\">\r\n" + //
+				"    <!ENTITY lol7 \"&lol6;&lol6;&lol6;&lol6;&lol6;&lol6;&lol6;&lol6;&lol6;&lol6;\">\r\n" + //
+				"    <!ENTITY lol8 \"&lol7;&lol7;&lol7;&lol7;&lol7;&lol7;&lol7;&lol7;&lol7;&lol7;\">\r\n" + //
+				"    <!ENTITY lol9 \"&lol8;&lol8;&lol8;&lol8;&lol8;&lol8;&lol8;&lol8;&lol8;&lol8;\">\r\n" + //
+				"]>\r\n" + //
+				"<lolz>&lol9;</lolz>";
+
+		Diagnostic diagnostic = d(14, 6, 14, 12, DTDErrorCode.EntityExpansionLimitExceeded, //
+				"The parser has encountered more than \"64 000\" entity expansions in this document; this is the limit imposed by the application.",

```

Comment:
Tests are failing. It seems like the error message should read
`\"64, 000\"`

Reply:
you need to set the locale for the test so that number formatting is platform independant

Reply:
Thanks for your suggestion @fbricon. Fixed.